_data/docs.yml

```diff
@@ -167,6 +167,17 @@ apis:
       legacy: 1
       stable: 1
       nightly: 1
+  rapidspdf:
+    name: RAPIDSMPF
```

Comment:
Is this the canonical product name? I see "RapidsMPF" (casing is different than RAPIDSMPF) on the README. Let's be consistent with the README, one way or the other.